### PR TITLE
Add OpInfo entry for alias_copy

### PR DESCRIPTION
**Pull request by rec**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #127232

cc @mrshenli @pritamdamania87 @zhaojuanmao @satgera @gqchen @aazzolini @osalpekar @jiayisuse @H-Huang @kwen2501 @awgu @penguinwu @fegin @XilunWu @wanchaol @fduwjj @wz337 @tianyu-l @wconstab @yf225 @chauhang @d4l3k @voznesenskym @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire

### CODE REVIEW

File: torch/_inductor/decomposition.py [hunk 1]
```diff
@@ -43,6 +43,7 @@
 inductor_decompositions = get_decompositions(
     [
         aten._adaptive_avg_pool2d_backward,
+        aten.alias_copy.default,
```
Comment (lezcano) [inline torch/_inductor/decomposition.py:46]: This decomposition can probably go into the default decompositons in `_decompositions/__init__.py` as it is not inductor-specfic.

File: torch/_decomp/__init__.py [hunk 2]
```diff
@@ -260,6 +260,7 @@ def core_aten_decompositions() -> Dict[torch._ops.OperatorBase, Callable]:
             aten.addcmul_,
             aten.addr,
             aten.affine_grid_generator,
+            aten.alias_copy.default,
```
Comment (lezcano) [inline torch/_decomp/__init__.py:263]:
```suggestion
            aten.alias_copy,
```
We are covering all overloads (default and out). This should fix the errors.

**Review by lezcano — APPROVED** [review]
Nice. Thank you!

**Comment by rec** [comment]
Thanks, I'll fix it post-haste! :-)

**Comment by rec** [comment]
Closed in favor of https://github.com/pytorch/pytorch/pull/128142, because I couldn't convince `ghstack` to resubmit the rebased commit...

**Pull request closed without merging**